### ISSUE TITLE
LN node crashed, please force close channel 633756303295971328 to node 02fd4115a3cf4e2d4f03f8947f2ba6366088d7a193fcd382b139b5a68d4531421e

### ISSUE THREAD

**Issue by vit001** [lightning-power-users/lightning-power-users-website#8]
Hello, I opened a LN channel to you, and subsequently my node crashed. I am unable to recover channel state. Can you please close channel 633756303295971328 to my node  02fd4115a3cf4e2d4f03f8947f2ba6366088d7a193fcd382b139b5a68d4531421e in order to un-stuck the stuck balances?
Thank you.

**Comment by PierreRochard** [comment]
I can force close it, but you won't be able to recover the funds using just your seed, you do need the channel state to recover even from a closed channel

**Comment by vit001** [comment]
Yes, I am realizing the funds may be lost permanently unfortunately... If
you have any ideas please let me know ...

**Comment by PierreRochard** [comment]
which version of LND were you using? We should find out exactly why it crashed
